File: src/main/java/top/dsbbs2/bukkitcord/nukkit/NukkitBootstrap.java
package top.dsbbs2.bukkitcord.nukkit;

import static top.dsbbs2.bukkitcord.api.PlatformManager.setBootstrapPlugin;

public class NukkitBootstrap extends NukkitPluginImpl {
    {
        setBootstrapPlugin(this);
    }
}

File: src/main/java/top/dsbbs2/bukkitcord/nukkit/NukkitBootstrap.kt
package top.dsbbs2.bukkitcord.nukkit

import top.dsbbs2.bukkitcord.api.PlatformManager.bootstrapPlugin

class NukkitBootstrap : NukkitPluginImpl() {
    init {
        bootstrapPlugin = this
        NukkitPluginImpl()
    }
}